apps/04TowersOfHanoi.js

```diff
@@ -19,24 +19,48 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
+function movePiece(startStack, endStack) {
   // Your code here
-
+  var start = stacks[startStack].pop();
+  stacks[endStack].push(start);
 }
 
-function isLegal() {
+function isLegal(startStack, endStack) {
   // Your code here
+var legal = false
+var startBlock = stacks[startStack][stacks[startStack].length -1]
+var endBlock = stacks[endStack][stacks[endStack].length - 1]
 
+if(stacks[endStack].length >=0){
```

Comment:
This isn't quite right. We don't want to set legal to true just because the end stack's length is 0 or greater.